region/lowernorfair/west.json

```diff
@@ -169,15 +178,51 @@
                   "name": "Acid Dive",
                   "notable": false,
                   "requires": [
-                    "canSuitlessLavaDive",
+                    "h_canNavigateHeatRooms",
+                    {"or": [
+                      "Gravity",
+                      "canSuitlessLavaDive"
+                    ]},
                     {"or": [
                       "HiJump",
-                      "canWalljump"
+                      "canWalljump",
+                      "canSpringBallJumpMidAir"
```

Comment:
Would this increase the frames by much?

Reply:
With one try of each walljump and springballjump I had the same acid frames count.  HiJump I got to be faster by doing it an unintuitive way.